CHANGES.rst

```diff
@@ -1,17 +1,21 @@
 ``zope.event`` Changelog
 ========================
 
-4.1.1 (unreleased)
+4.2.0 (unreleased)
 ------------------
 
-- Claim support for Python 3.5.
+- Add support for Python 3.5.
```

Comment:
FWIW I say "claim" when no changes need to be done (other than adding the new version to .travis.yml/tox.ini/setup.py classifiers), and "add" when something actually breaks and needs to be fixed to make it work on the new Python version.

Probably nobody besides me cares about this subtle distinction.